### PR TITLE
Software 0.28.7 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: software_versioned_docs/version-0.28/release-notes.md [hunk 1]
```diff
@@ -21,6 +21,16 @@ This document includes all release notes for Astronomer Software v0.28.
 
 All Astronomer Software versions are tested for scale, reliability and security on Amazon EKS, Google GKE, and Azure AKS. If you have questions or an issue to report, contact [Astronomer support](https://support.astronomer.io).
 
+## v0.28.7
+
+Release date: October 14, 2022
```
Comment (jwitz) [inline software_versioned_docs/version-0.28/release-notes.md:26]:
```suggestion
Release date: October 13, 2022
```

File: software_versioned_docs/version-0.28/release-notes.md [hunk 2]
```diff
@@ -21,6 +21,16 @@ This document includes all release notes for Astronomer Software v0.28.
 
 All Astronomer Software versions are tested for scale, reliability and security on Amazon EKS, Google GKE, and Azure AKS. If you have questions or an issue to report, contact [Astronomer support](https://support.astronomer.io).
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.28/release-notes.md:22]:
```suggestion
All Astronomer Software versions are tested for scale, reliability, and security on Amazon EKS, Google GKE, and Azure AKS. If you have questions or an issue to report, contact [Astronomer support](https://support.astronomer.io).
```

File: software_versioned_docs/version-0.28/release-notes.md [hunk 3]
```diff
@@ -21,6 +21,16 @@ This document includes all release notes for Astronomer Software v0.28.
...
 
+## v0.28.7
+
+Release date: October 13, 2022
```
Comment (vishwas-astro) [inline software_versioned_docs/version-0.28/release-notes.md:26]: Release date should be 14th.

**Review by vishwas-astro — APPROVED** [review]

File: software_versioned_docs/version-0.28/release-notes.md (hunk 3 above)
Comment (jwitz) [inline software_versioned_docs/version-0.28/release-notes.md:26]:
```suggestion
Release date: October 14, 2022
```

**Pull request merged**